source/_includes/pup40_platforms_redhat_like.markdown

```diff
@@ -1,10 +1,10 @@
 We publish official packages and run automated testing on the following versions for Puppet 4.x:
 
-* Enterprise Linux 6 (also supported by [Puppet Enterprise][peinstall])
 * Enterprise Linux 7 (also supported by [Puppet Enterprise][peinstall])
-
+* Enterprise Linux 6 (also supported by [Puppet Enterprise][peinstall])
+* Enterprise Linux 5 (also supported by [Puppet Enterprise][peinstall])
```

Comment:
Hmm, wait, we do have el5 server packages (puppetserver and puppetdb). I'm trying to remember if that was a mistake or intentional... I'm fairly sure that was a mistake...